.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+<!--
+    We categorize our issues into 2 categories:
+    * Bug: Prefix your issue title with [BUG]
+    * Feature Request: Prefix your issue title with [FEATURE]
```

Comment:
I don't think people are honestly going to follow this - and I don't think it's important. We'll use a label to categorise them.

No prefixing please.